### ISSUE TITLE
bug: unknown service proto.Destination

### ISSUE THREAD

**Issue by nicosuave** [cloudquery/cloudquery#12105]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am following along with the Postgres -> Motherduck example and I see

```
Error: failed to sync v1 source postgresql: rpc error: code = Unimplemented desc = unknown service proto.Destination
```

### Expected Behavior

Seems like some kind of serious internal Cloudquery failure?

### CloudQuery (redacted) config

```yaml
spec:
  name: "postgresql"
  path: "cloudquery/postgresql"
  version: "v1.2.1"
  destinations: ["motherduck"]
  tables: ["*"]
  spec:
    connection_string: "postgresql://..."
    cdc: false
---
kind: destination
spec:
  name: "motherduck"
  version: "v4.1.1"
  path: "cloudquery/duckdb"
  write_mode: "overwrite-delete-stale"
  migrate_mode: "safe"
  spec:
    connection_string: "md"
```

### CloudQuery (redacted) logs

```
Error: failed to sync v1 source postgresql: rpc error: code = Unimplemented desc = unknown service proto.Destination
```

### CloudQuery version

cloudquery version 3.9.0

### Additional Context

Neon Postgres as the source

### Pull request (optional)

- [ ] I can submit a pull request

**Comment by bbernays** [comment]
@nicosuave - Thank you for opening this issue! We have released duckdb `v4.1.2` with a fix for your issue. If this doesn't resolve your issue please reopen this issue!

**Comment by nicosuave** [comment]
Much thanks @bbernays that did the trick
